utils/utils.go

```diff
@@ -2,58 +2,12 @@ package utils
 
 import (
 	"crypto/ecdsa"
-	"encoding/json"
 	"errors"
-	"log"
-	"math/big"
-	"os"
-	"path/filepath"
-	"regexp"
-
 	gethcommon "github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/crypto"
-	"gopkg.in/yaml.v3"
+	"math/big"
 )
 
-func ReadFile(path string) ([]byte, error) {
-	b, err := os.ReadFile(filepath.Clean(path))
-	if err != nil {
-		return nil, err
-	}
-	return b, nil
-}
-
-func ReadYamlConfig(path string, o interface{}) error {
```

Comment:
These methods are used in few of libraries we import eigensdk in. So let's not remove

Reply:
Aaah makes sense. Also, can we move these functions to a different module like `eigenUtils` because intuitively `utils` are for internal use?

Reply:
ideally all internal stuff would've been in an `internal` package but we didn't follow that convention.

I am not sure what the ideal way of doing this. Are you suggesting we can have `utils` and `eigenUtils` both ? one for internal and one for external?

Reply:
Yep, this might cause breaking changes for the SDK users though.
Honestly, I don't see what these utils functions really abstract away for users. I believe the SDK should be used for its interfaces and the utils like reading config files should be handled by client apps.
What do you think?

Reply:
yea you are right. the only reason I added is that, multiple repository we have use this function and I just wanted to have it once. but I kind of agree it's on client to have these standard utils on their own. they can choose their own way to do this.

Reply:
Let me know the PoA for this PR @shrimalmadhur!

Reply:
Lets remove these function. I will make sure dependencies take care of it. @samlaf you good with that?

Reply:
Yep

Reply:
@RohanShrothrium feel free to make these changes

Reply:
@shrimalmadhur if you don't mind can you resolve all the comments which do not need any change? I am a bit confused as to what functions should be added back.

Reply:
done

Reply:
This function is removed as well in the initial commit ser!

Reply:
@shrimalmadhur is this good to be merged after we rebase on master?

Reply:
yep. @RohanShrothrium can you rebase and resolve conflicts? and then we can merge this